code/modules/mob/living/carbon/human/species/station/station_ch.dm

```diff
@@ -38,4 +38,32 @@
 
 /datum/species/vox
 	spawn_flags = SPECIES_CAN_JOIN | SPECIES_IS_WHITELISTED | SPECIES_WHITELIST_SELECTABLE
-*/
\ No newline at end of file
+*/
+
+//Can use digitigrade flags
```

Comment:
Are Xenochimera not allowed to use Digitigrade legs, or do they go off the sprite choice under "sprite base"?

Reply:
Added in commit below.